libs/audit-service/src/main/java/uk/gov/di/ipv/core/library/auditing/AuditEvent.java

```diff
@@ -70,4 +71,22 @@ public AuditEvent(
             AuditRestricted restricted) {
         this(eventName, componentId, user, null, restricted);
     }
+
+    public static AuditEvent createAuditEventWithDeviceInformation(
```

Comment:
If they're in this class, you could probably drop duplicate `AuditEvent` - e.g. `AuditEvent.createWithDeviceInformation`, but whatever you think is clearest :)